packages/microcosm-www-next/src/stylesheets/structure/_section.scss

```diff
@@ -171,7 +171,7 @@ $graphic-width: calc(50vw - #{$page-gutter-desktop});
   max-width: 600px;
   opacity: 0.33;
   position: relative;
-  transition: opacity 0.7s;
+  transition: transition(opacity);
```

Comment:
does `transition()` set the timing everywhere to keep things consistent?

Reply:
yup yup